commit 8e3ac2b9e8a9bea4978c0a921495ec284f4371fb
Author: popcornmix <popcornmix@gmail.com>
Date:   Fri Sep 29 16:15:01 2017 +0100

    vcsm: Fix up macros to avoid breaking numbers used by existing apps

diff --git a/drivers/char/broadcom/vc_sm/vc_sm_knl.h b/drivers/char/broadcom/vc_sm/vc_sm_knl.h
index e50fd8eaecef..4f120421c2d1 100644
--- a/drivers/char/broadcom/vc_sm/vc_sm_knl.h
+++ b/drivers/char/broadcom/vc_sm/vc_sm_knl.h
@@ -29,8 +29,8 @@ enum vc_sm_lock_cache_mode {
 
 /* Cache functions */
 #define VCSM_CACHE_OP_INV       0x01
-#define VCSM_CACHE_OP_FLUSH     0x02
-#define VCSM_CACHE_OP_CLEAN     0x03
+#define VCSM_CACHE_OP_CLEAN     0x02
+#define VCSM_CACHE_OP_FLUSH     0x03
 
 /* Allocate a shared memory handle and block. */
 int vc_sm_alloc(struct vc_sm_alloc_t *alloc, int *handle);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1136_8e3ac2b9e8a9bea4978c0a921495ec284f4371fb.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1136_8e3ac2b9e8a9bea4978c0a921495ec284f4371fb.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
